engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PassPersistance.java

```diff
@@ -143,7 +143,6 @@ protected Graph readObject(Input in) throws IOException {
 
       var nextIdCounter = in.readInt();
       var g = new Graph(rootScope, nextIdCounter, links);
-      g.freeze();
```

Comment:
Why remove `g.freeze()` from here? Is it because of failing tests? Is it not needed anymore?

Reply:
Yes, this is weird.

Reply:
> Is it because of failing tests? 

No, it is not because of a test. Rather:
- #11027
- #11022

> Is it not needed anymore?

`freeze`ing  doesn't work with `DebugEvalNode` right now.